content/blog/cfp.md

```diff
@@ -1,16 +1,78 @@
 ---
```

Comment:
no need for a blog post, remove this file

Reply:
Done